drivers/telescope/lx200gemini.cpp

```diff
@@ -442,17 +444,16 @@ bool LX200Gemini::ReadScopeStatus()
             SlewRateS[SLEW_CENTERING].s = ISS_ON;
             IDSetSwitch(&SlewRateSP, nullptr);
 
-            TrackState = SCOPE_TRACKING;
             LOG_INFO("Slew is complete. Tracking...");
         }
     }
     else if (TrackState == SCOPE_PARKING)
     {
         updateParkingState();
-        updateMovementState();
 
         if (isSlewComplete())
         {
+            LOG_DEBUG("Park is complete ...");
             SetParked(true);
             sleepMount();
```

Comment:
sleepMount() needs to be commented/removed for the mount not to go into SCOPE_SLEWING after SCOPE_PARKING

Reply:
Sorry, let me be more explicit:
The initial commit fixed the issue were the mount went back to tracking after parking completed, thus preventing further actions (like a flat sequence).
However the status in ekos and mount control showed "Slewing" instead of "Parked".

To reproduce, I simply unparked the mount set tracking on, manually slew a bit and then hit park.
Status went: Tracking->Parking->Slewing

After further investigation I found out sleepMount() was causing this (The mount itself was reporting SLEWING after Gv# command). Once sleepMount() was commented the status correctly displayed "Parked" after parking.

I tested further starting from parked state with a sequence of 10 lights followed by 10 flats (with park mount checked). It worked perfectly, mount unparked, slewed, tracked, then parking and parked.